fabtests/test_configs/lnx/lnx.exclude

```diff
@@ -0,0 +1,40 @@
+# Regex patterns of tests to exclude in runfabtests.sh
+
+# Unsupported EP types
+^fi_msg
+-e msg
+^fi_dgram
+-e dgram
+cm_data
+
+# Exclude tests with unsupported capabilities
+atomic
+scalable_ep
+shared_ctx
+
+# Uses SOCKADDR
+av_test
+
+# Uses FI_MSG
+multinode
+rdm_pingpong
+multi_recv
+cntr_pingpong
+multi_client
+flood
+multi_ep
+cq_data
+av_xfer
+shared_av
+fi_rdm\b
```

Comment:
what is the `\b` here?

Reply:
\b matches a word boundary (includes spaces and end of string) so that matches to any test that references just the fi_rdm test ("fi_rdm" and "fi_rdm -U") which requires FI_MSG which lnx currently does not support. Without the \b it would match anything with "fi_rdm" in it which includes things like "fi_rdm_tagged_pingpong" which we don't want to exclude